packages/app/src/components/uiBlocks/Button.vue

```diff
@@ -10,7 +10,8 @@
         class="my-auto font-heading font-extralight"
         :class="{
           'ml-4': icon,
-          'mx-auto': (!rounded && !icon) || textCenter,
+          'mx-auto': !rounded && !icon,
+          'w-full text-center': textCenter,
```

Comment:
Did you checked if that changes the style of all the other buttons?

Reply:
seems to be fine